moon-store-mac-os/moon-store-mac-os/Helpers/Extensions/Data+String.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  Data+String.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 12/8/24.
+//
+
+import Foundation
+
+extension Data {
+    mutating func appendString(_ string :String) {
```

Comment:
Cambiar el nombre y agregar correcta identacion
```suggestion
    mutating func appendString(_ string: String) {
```